client/src/components/Navbar.js

```diff
@@ -36,23 +37,33 @@ const SearchBar = styled.input`
 export const Navbar = () => {
   return (
     <nav className="nav">
-      <Title>Project Metroid</Title>
+      <Title className="title">Project Metroid</Title>
+      <button className="mobile-search-toggle nav__mobile--link">
+        <i class="fas fa-search"></i>
```

Comment:
For now, using font-awesome should be ok but as our app grows we should think about using SVGS instead as it is more scalable and light weight.